config/init_maria_db.sql

```diff
@@ -0,0 +1,104 @@
+# This file is executed immediately after mysql_install_db,
+# to initialize a fresh data directory.
```

Comment:
Is this separate file created for the super_read_only changes which are only present in the MySQL file?